cmd/flux/cmd/fmt.go

```diff
@@ -64,6 +66,12 @@ func format(script string) error {
 		return fmt.Errorf("failed to format the query: %s, %v", script, err)
 	}
 
+	if analyzeCurrentDirectory {
+		if string(fromFile) != formatStr {
+			return fmt.Errorf("commit includes flux files that are not fluxfmt-ed")
```

Comment:
Glad  the actual work here was so small.

Reply:
nit, but `flux fmt -c` isn't always run in the context of git or checking a specific commit. Could we make this error message more generic? Something like a list of files that are not formatted or just a statement that a unformatted file was found?

Reply:
Yes I realise the same and added that in the code. Thanks.